app/containers/MessageBox/index.js

```diff
@@ -818,56 +903,120 @@ class MessageBox extends Component {
 			<>
 				<CommandsPreview commandPreview={commandPreview} showCommandPreview={showCommandPreview} />
 				<Mentions mentions={mentions} trackingType={trackingType} theme={theme} />
-				<View style={[styles.composer, { borderTopColor: themes[theme].separatorColor }]}>
-					<ReplyPreview
-						message={message}
-						close={replyCancel}
-						username={user.username}
-						replying={replying}
-						getCustomEmoji={getCustomEmoji}
+				{fullScreenUp ? (
```

Comment:
This now feels a bit complicated with logic and a few fragment blocks. Could it be split into different components? It feels like `renderFullScreenComposer` could be converted into one

Reply:
I was thinking about it, thanks for the idea I'll try to convert this function to component.